scripts/valhalla_build_config

```diff
@@ -534,6 +541,13 @@ help_text = {
     'max_timedep_distance': 'Maximum b-line distance between locations to allow a time-dependent route',
     'max_alternates': 'Maximum number of alternate routes to allow in a request',
     'max_exclude_polygons_length': 'Maximum total perimeter of all exclude_polygons in meters'
+  },
+  'metrics': {
+    'host': 'The statsd host address',
+    'port': 'The statsd port',
+    'prefix': 'The statsd prefix to use for each metric',
+    'batch_size': 'Approximate maximum size in bytes of each batch of stats to send to statsd',
```

Comment:
How does udp vs tcp factor in here? If we're shipping to a udp socket, I imagine you dont want to go above the MTU size (1476 byte?).

Reply:
it only supports udp. max size for ethernet mtu is about 1500 as you suggest. i am setting the default to 1024 to stay below that thought i highly doubt that we'll ever send so many metrics in one go that we'd exceed that